docs/features.md

```diff
@@ -0,0 +1,88 @@
+# Features
+
+This document lists and briefly explains the functionality offered by
+Apisonator.
+
+- [Authorization of applications](#authorization-of-applications)
+- [Rate-limit](#rate-limit-of-applications)
+- [Reporting of usage stats](#aggregation-of-usage-stats)
+- [Reporting of response codes](#aggregation-of-response-codes)
+- [Alerts](#alerts)
+- [Events](#events)
+- [Real-time traffic](#real-time-traffic)
+- [Integration errors](#integration-errors)
+- [Usage metrics of the Porta master account](#usage-metrics-of-the-porta-master-account)
+- [Internal management API](#internal-management-api)
+
+## Authorization of applications
+
+Requests are authorized based on some credentials that identify a service
+(provider key, service token) and some credentials that identify an application
+(app ID + app key, user key, oauth token).
+
+## Rate-limit of applications
+
+Rate limits are applied per application. The limits can be per minute, hour,
+day, week, month, year, and "eternity". The limits are applied in calendar
+periods of time. So for example, if the limit is per year and the year is 2020,
+when applying the limit, Apisonator will only take into account the counters
+starting from Jan 1st 2020, not from a year ago.
+
+## Aggregation of usage stats
+
+Apisonator aggregates usage stats by the same periods used for rate-limiting
+(except for minute). It aggregates data by service and also by application. This
+allows external system to query the usage for a given month, year, etc.
+efficiently.
+
+## Aggregation of response codes
+
+Apisonator also aggregates response codes using the same periods used to
+aggregate usage stats. These response codes are the ones returned by the
+upstream API and are sent to Apisonator via APIcast or other integration
+mechanism (Istio adapter, Java plugin, etc.).
+
+## Alerts
+
+Apisonator sends usage alerts to Porta via a webhook. This alerts can be
+configured to be triggered when an application reaches 50% of it usage quota,
+80%, 90%, etc.
```

Comment:
Fixed thresholds - or he thresholds are configurable?
How many? Unlimited number of thresholds.

Thresholds on any metric or just hits?

No thresholds on response codes...